### ISSUE TITLE
White page with a "2024" when I navigate through my site

### ISSUE THREAD

**Issue by momocode-de** [symfony/ux#1632]
Sometimes it happens that when I navigate through my (local) site, only a white page with a "2024" appears. The following error message then appears in the console:

> Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.

![image](https://github.com/symfony/ux/assets/57801067/0708c55b-c9a5-4115-be53-01b2632d31ae)

I then updated turbo from 8.0.3 to 8.0.4. Now the error message no longer appears in the console, but the white page with the "2024" still appears regularly, without an error message. Does anyone have any idea why this is happening?

So far I use a few stimulus controllers, a few Twig Live Components and custom Javascript, which is integrated into the app.js (added to the template via importmap).

**Comment by smnandre** [comment]
That seems to me like a Turbo bug ... maybe related to this ? https://github.com/hotwired/turbo/pull/1225

**Comment by momocode-de** [comment]
@smnandre I'm not sure. Is it possible to deactivate the "Single Page" behavior of Turbo and still use Stimulus and Live components? I have noticed that the single page behavior is active as soon as the following line is in the bootstrap.js: 
`import { startStimulusApp } from '@symfony/stimulus-bundle';`

If I remove the line, the page navigations happen normally and I no longer get a blank page. But then my stimulus controllers won't work either.

**Comment by momocode-de** [comment]
I have found the problem. I had the following code in my footer:
```
Copyright © <script>
    document.write(new Date().getFullYear())
</script>
```
I had just copied that from somewhere and didn't realize it was in there. As described [here](https://symfony.com/bundles/ux-turbo/current/index.html#1-make-sure-your-javascript-is-turbo-ready), every `<script>` tag should be in the `<head>` and that was the problem. I have now replaced the year's output with Twig code and now the problem has been solved.
